client/client.go

```diff
@@ -341,7 +341,7 @@ func addChange(cl *changelist.FileChangelist, c changelist.Change, roles ...stri
 
 // AddTarget creates new changelist entries to add a target to the given roles
 // in the repository when the changelist gets applied at publish time.
-// If roles are unspecified, the default role is "targets".
+// If roles are unspecified, the default role is data.CanonicalTargetsRole.
```

Comment:
nit: this could stay as "targets" since it's just a comment, but this is fine too